packages/vite-plugin-cloudflare/src/cloudflare-environment.ts

```diff
@@ -1,4 +1,5 @@
 import * as vite from 'vite';
+import { nodeBuiltIns } from './node-builtin-modules';
```

Comment:
Why not use:

```
import { builtinModules } = "node:modules";

const nodeBuiltIns = builtinModules.concat(builtinModules.map((m) => `node:${m}`);
```

Reply:
What do you mean with `import { builtinModules } = "node:modules";`? 😮 

if you mean just exporting the non-prefixed modules in `node-builtin-modules` and do the prefixing using `concat` that sounds find by me 🙂

Reply:
@petebacondarwin is this what you meant? 🙂  https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/37/commits/7aefe02ec78409c19893a2a52b03763c40064ea7

Reply:
Sorry there was a typo in my suggestion. This is my suggestion:

```suggestion
import { builtinModules } from "node:modules";

const nodeBuiltIns = builtinModules.concat(builtinModules.map((m) => `node:${m}`);
```

Reply:
Ah ok awesome! I wasn't aware of this API, I've applied the change, thanks! 😄